packages/kit/src/views/Market/MarketDetailV2/components/SwapPanel/SwapPanelContent.tsx

```
@@ -9,13 +11,15 @@ import { AntiMEVToggle } from './components/AntiMEVToggle';
 import { ApproveButton } from './components/ApproveButton';
 import { BalanceDisplay } from './components/BalanceDisplay';
 import { SlippageSetting } from './components/SlippageSetting';
-import { TokenInputSection } from './components/TokenInputSection';
+import { TokenInputSection, type ITokenInputSectionRef } from './components/TokenInputSection';
```

ESLint

[error] Replace `·TokenInputSection,·type·ITokenInputSectionRef·` with `⏎··TokenInputSection,⏎··type·ITokenInputSectionRef,⏎`

(prettier/prettier)